models/list_option/list_option.go

```diff
@@ -48,16 +48,16 @@ func FindByAction(action string) *ListOptionStruct {
 	table := interfaceDB.Init()
 	defer table.Close()
 
-	var listOptionId int64
-	err := table.QueryRow("select id from list_options where action = ?;", action).Scan(&listOptionId)
+	var listOptionID int64
+	err := table.QueryRow("select id from list_options where action = ?;", action).Scan(&listOptionID)
 	if err != nil {
 		logging.SharedInstance().MethodInfo("ListOption", "FindByAction").Info("cannot find list_option")
 		return nil
 	}
-	return NewListOption(listOptionId, action)
+	return NewListOption(listOptionID, action)
 }
 
-func FindById(id sql.NullInt64) *ListOptionStruct {
+func FindByID(id sql.NullInt64) *ListOptionStruct {
```

Comment:
__[golint]__
* [golint] exported function FindByID should have comment or be unexported